README.md

```diff
@@ -362,6 +363,7 @@ _[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software_
   * [Pump.io](http://pump.io/) - Stream server that does most of what people really want from a social network. ([Source Code](https://github.com/e14n/pump.io)) `Apache` `Nodejs`
   * [Reddit](https://www.reddit.com/) - Social networking and news website where registered community members can submit content. ([Demo](https://www.reddit.com/), [Source Code](https://github.com/reddit/reddit)) `CPALv1` `Python`
   * [RedMatrix](https://github.com/redmatrix/redmatrix) - Complete decentralised publishing, sharing, and communications system. `MIT` `PHP`
+  * [Symphony](https://github.com/b3log/symphony) - A modern community (forum/SNS/blog) platform written in Java. ([Demo](https://hacpai.com), [Source Code](https://github.com/b3log/symphony)) `GPLv3` `Java`
```

Comment:
Please remove the Source Code link as it is the same as the homepage URL and duplicate links are not allowed.

     * [Symphony](https://github.com/b3log/symphony) - A modern community (forum/SNS/blog) platform written in Java. ([Demo](https://hacpai.com)) `GPLv3` `Java`